src/families/cosmos/SendRowsFee.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import React, { useCallback } from "react";
+import { View, StyleSheet, Linking } from "react-native";
+import type { AccountLike } from "@ledgerhq/live-common/lib/types";
+import { Trans } from "react-i18next";
+import type { Transaction } from "@ledgerhq/live-common/lib/families/cosmos/types";
+import {
+  getAccountUnit,
+  getAccountCurrency,
+} from "@ledgerhq/live-common/lib/account";
+import SummaryRow from "../../screens/SendFunds/SummaryRow";
+import LText from "../../components/LText";
+import CurrencyUnitValue from "../../components/CurrencyUnitValue";
+import CounterValue from "../../components/CounterValue";
+import ExternalLink from "../../icons/ExternalLink";
+import { urls } from "../../config/urls";
+import colors from "../../colors";
+
+type Props = {
+  account: AccountLike,
+  transaction: Transaction,
+};
+
+export default function CosmosFeeRow({ account, transaction }: Props) {
+  const extraInfoFees = useCallback(() => {
+    Linking.openURL(urls.feesMoreInfo);
+  }, []);
+
+  if (transaction.family !== "cosmos") return null;
```

Comment:
shouldn't this be an invariant?
also why is there cosmos in the PR of algo?

Reply:
I figured while integrating Algo that the fees were missing in the send summary for Cosmos